pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -234,6 +240,13 @@ func WithPodMaxInUnschedulablePodsDuration(duration time.Duration) Option {
 	}
 }
 
+// WithEnqueuePluginMap sets preEnqueuePluginMap for PriorityQueue.
+func WithEnqueuePluginMap(m map[string][]framework.PreEnqueuePlugin) Option {
```

Comment:
You might want to rename it to `WithPreEnqueuePluginMap` because we named the interface `PreEnqueue`.

Reply:
done.